src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+
+  if (days >= 7) {
+    return days * dayPrice - 50;
```

Comment:
Replace the '7' and '50' with constants that describe these values. For example, 'LONG_TERM_RENTAL' and 'LONG_TERM_DISCOUNT'.